app/celery/contact_information_tasks.py

```diff
@@ -31,21 +35,12 @@ def lookup_contact_info(self, notification_id):
                 f"{notification.notification_type} is not supported")
 
     except VAProfileRetryableException as e:
-        current_app.logger.exception(e)
-        try:
-            self.retry(queue=QueueNames.RETRY)
-        except self.MaxRetriesExceededError:
-            message = (
-                'RETRY FAILED: Max retries reached. '
-                f'The task lookup_contact_info failed for notification {notification_id}. '
-                'Notification has been updated to technical-failure'
-            )
-
-            update_notification_status_by_id(
-                notification_id, NOTIFICATION_TECHNICAL_FAILURE, status_reason=e.failure_reason
-            )
-            raise NotificationTechnicalFailureException(message) from e
-
+        if can_retry(self.request.retries, self.max_retries):
+            current_app.logger.warning("Unable to get contact info for notificaiton id: %s, retrying", notification_id)
```

Comment:
spelling: change 'notificaiton' to 'notification'

Reply:
Fixed